tools/deployment/k8s/s3/sample-resource-files/deploy-yscope-log-viewer.yaml

```
@@ -0,0 +1,32 @@
+        - name: LOG_VIEWER_BUCKET
+          value: log-viewer
+        - name: RELEASE_URL
+          value: https://github.com/y-scope/yscope-log-viewer/releases/download/v0.1.0-main%2B20250523.fe22a3c/dist-0.1.0-main+20250523.fe22a3c.tar.gz
+      volumeMounts:
+        - name: secret-volume
+          mountPath: "/root/.aws"
+      imagePullPolicy: IfNotPresent
+  volumes:
+    - name: secret-volume
+      secret:
+        secretName: aws-credential
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)